gradle/libs.versions.toml

```diff
@@ -1,18 +1,16 @@
 [versions]
 kotlin = "1.7.10"
-save-core = "0.3.2"
+save-core = "0.3.3"
 ktor = "2.1.0"
 okio = "3.2.0"
-serialization = "1.3.3"
+serialization = "1.4.0"
 kotlinx-datetime = "0.4.0"
 kotlinx-coroutines = "1.6.4"
-kotlin-wrappers = "1.0.0-pre.369"
+kotlin-wrappers = "1.0.0-pre.373"
 spring = "5.3.22"
-spring-boot = "2.7.2"
-spring-security = "5.7.2"
+spring-boot = "2.7.3"
 spring-cloud = "3.1.3"
 spring-cloud-kubernetes = "2.1.3"
-kafka-client = "3.2.0"
```

Comment:
discussed offline with @icemachined -- we decided to use latest version of kafka client and override version from spring-boot-dependencies